targets/syslog_test.go

```diff
@@ -48,6 +49,7 @@ func ExampleSyslog() {
 }
 
 func TestSyslogPlain(t *testing.T) {
+	t.Skip("Skipping to debug why is failing on CI")
```

Comment:
Local syslog is probably not available in test container.

Reply:
humm, good to know, i will check that! thanks for the insight and for the review

Reply:
added the test back :)